README.md

```diff
@@ -71,35 +71,17 @@ abstractions or just feel ad hoc. Nickel buys you more for less.
 
 ## Getting started
 
-### Build
-
-[rust-guide]: https://doc.rust-lang.org/cargo/getting-started/installation.html
-
-1. Download build dependencies:
-   - **With Nix**: If you have [Nix](https://nixos.org/nix) installed:
-     ```
-     $ nix-shell shell.nix
-     ```
-     to be dropped in a shell, ready to build.
-   - **Without Nix**: otherwise, follow [this guide][rust-guide] to install Rust
-     and Cargo first.
-2. Build Nickel:
-   ```
-   $ cargo build
-   ```
-   And voilà! Generated files are placed in `target/debug`.
-
 ### Run
 
-1. *(optional)* make a symbolic link to the executable:
-  ```
-  $ ln -S nickel target/debug/nickel
-  ```
-  Alternatively, you can use `cargo run` to launch nickel. To pass arguments to
-  nickel using `cargo run`, use `--`:
-  ```
-  $ cargo run -- -f program.ncl
-  ```
+1. Start Nickel either
+   * with [flake-enabled](https://nixos.wiki/wiki/Flakes) Nix directly from
+     this repo with `nix run github:tweag/nickel`, passing in arguments with
```

Comment:
Actually you don't even have to be in the repo or to clone it beforehand with flakes. Maybe we should make this clearer.

Reply:
BTW, `nix run nickel` also works, since `nickel` is in the global flake registry :-)

Reply:
Dam, I'm glad that I started watching this repo. I completely forgot about this feature :D Good stuff :rocket: